### PR TITLE
feat: additional border props and  custom style

### PR DESCRIPTION
**Pull request by AlanRad-QH**
## Description

Additional border props for `Box` component to style autocomplete location list

💁‍

## Related issues

[](https://app.shortcut.com/qantasaccommodation/story/113070/change-property)

💪

## Looks like this

N/A

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've checked if you need to update any relevant typescript declaration file(s), for example component props (including styled-system props) or theme.
- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan), [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: src/components/Box/Box.jsx [hunk 1]
```diff
@@ -44,6 +38,46 @@ const flexGrow = style({
   cssProperty: 'flexGrow',
 });
 
+const borderBottomColor = style({
```
Comment (ajlu) [inline src/components/Box/Box.jsx:41]: can we also update `Box.test.jsx` to test all these new props?

Comment (AlanRad-QH) [reply src/components/Box/Box.jsx:41]: Good point! Git it. Thanks.

**Review by ajlu — APPROVED** [review]
🚀

**Review by philipwindeyer — APPROVED** [review]
🚢

**Pull request merged**